common/Timer.h

```diff
@@ -30,8 +30,10 @@ class Timer {
     // Don't report timer when it gets destructed.
     void cancel();
 
+    void setTag(ConstExprStr name, ConstExprStr value);
```

Comment:
Like metric names, tags must also be `ConstExprStr` to make it more difficult to have metric cardinality issues.